src/main/resources/application.yaml

```diff
@@ -15,9 +15,9 @@ spring:
   application:
     name: Send Letter Provider
   datasource:
-      url: jdbc:postgresql://${LETTER_TRACKING_DB_HOST:localhost}:${LETTER_TRACKING_DB_PORT:5432}/${LETTER_TRACKING_DB_NAME:letter-tracking}
-      username: ${LETTER_TRACKING_DB_USER_NAME:postgres}
-      password: ${LETTER_TRACKING_DB_PASSWORD:postgres}
+      url: jdbc:postgresql://${LETTER_TRACKING_DB_HOST:send-letter-producer-database}:${LETTER_TRACKING_DB_PORT:5432}/${LETTER_TRACKING_DB_NAME:letter_tracking}
```

Comment:
can we keep the db name `letter-tracking` consistent with `application.yml`

Reply:
we can as soon as i am contra-argued hyphens are recommended for database/table names. foolishly created one in the past and it caused few problems because it is not the usual naming convention

Reply:
You can create a database name that contains non alphanumeric characters such as the underscore _ and dash.Not blocking the merge because of this.